lxd/util/net.go

```diff
@@ -175,3 +176,26 @@ func IsAddressCovered(address1, address2 string) bool {
 
 	return false
 }
+
+// SysctlGet retrieves the value of a sysctl file in /proc/sys.
+func SysctlGet(prefix string, path string) (string, error) {
```

Comment:
@keyallis please can the signature for this function just be `SysctlGet(path string)` and then update the usage to prefix with `net/` rather than have the extra argument.

Reply:
I've addressed this and it's usage in the most recent commit